input.txt

```diff
@@ -1 +1 @@
-﻿
\ No newline at end of file
+﻿Test is in progress but it is not finished yet. Please be patient. Thank you.
```

Comment:
Nice input data, love it :)

Reply:
Is it ? :) Is it covering at least basic functionality ? Have you tried running it locally ?